Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs

```diff
@@ -142,14 +155,17 @@ private void Detach()
                     {
                         continue;
                     }
+                    double vibeSpeed = (_lastVibration.LeftMotorSpeed + _lastVibration.RightMotorSpeed) / (2.0 * 65535.0);
                     await _bpServer.SendMessage(new SingleMotorVibrateCmd(device.Index,
-                        (_lastVibration.LeftMotorSpeed + _lastVibration.RightMotorSpeed) / (2.0 * 65535.0)));
+                        vibeSpeed * _vibrationMultiplier > 1 ? 1 : vibeSpeed * _vibrationMultiplier));

```

Comment:
Let's simplify this a bit and just make it Math.Min(vibeSpeed * _vibrationMultiplier, 1.0)

Reply:
Totally forgot that Math.Min existed... And now I probably never will again (baring any more 3am coding)